docs/docs/recipes/transforming-data.md

```diff
@@ -54,3 +54,65 @@ export const query = graphql`
 
 - [Tutorial on transforming Markdown to HTML](/tutorial/part-six/#transformer-plugins) using `gatsby-transformer-remark`
 - Browse available transformer plugins in the [Gatsby plugin library](/plugins/?=transformer)
+
+## Transforming images into grayscale using GraphQL
+
+### Prerequisites
+
+- A Gatsby site with `gatsby-config.js` and an `index.js` page
+- The `gatsby-transformer-sharp` plugin installed
+- The `gatsby-plugin-sharp` plugin installed
+- A source plugin installed, such as `gatsby-source-filesystem`
+- An image (`.jpg`, `.png`, `.gif`, `.svg`, etc.) in the `src` folder
+
+### Directions
+
+1. Make sure that the config file has the correct information.
```

Comment:
Users aren't going to know what makes config file information correct, so you need to guide them. It would be more successful to explain what's in the following code example:

"Edit your `gatsby-config.js` file to source images for Gatsby's GraphQL data layer. A common approach is to source them from an images directory using the `gatsby-source-filesystem` plugin:"

Reply:
Thank you!! I've added this in.